code/notificationbuilder/src/main/res/layout/push_template_horizontal_catalog.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/catalog_container_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:clipToPadding="false"
+    android:clipChildren="false"
+    android:orientation="horizontal"
+    android:theme="@style/DayNightTheme">
+
+    <ImageView
+        android:id="@+id/product_image"
+        android:scaleType="fitXY"
+        android:adjustViewBounds="true"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/product_catalog_image_height"
```

Comment:
Can we not specify height here and instead set it to 0? In small devices the thumbnails might get cropped if this specified height is too much. We can have it take whatever space is left after all the other views are created

Reply:
the image is not shown if the height is set to 0dp